include/secp256k1_schnorrsig.h

```diff
@@ -104,6 +106,25 @@ SECP256K1_API SECP256K1_WARN_UNUSED_RESULT int secp256k1_schnorrsig_verify(
     const secp256k1_xonly_pubkey *pubkey
 ) SECP256K1_ARG_NONNULL(1) SECP256K1_ARG_NONNULL(2) SECP256K1_ARG_NONNULL(3) SECP256K1_ARG_NONNULL(4);
 
+SECP256K1_API int secp256k1_schnorrsig_aggregate(
+    const secp256k1_context* ctx,
+    unsigned char* aggsig,
+    size_t* aggsig_size,
+    unsigned char **sig64,
+    unsigned char **msg32,
+    secp256k1_xonly_pubkey *pubkey,
+    uint32_t n_sigs
```

Comment:
I know C is sloppy with this, but const unsigned char **sig64 would be nice.  And msg32.

Also, n_sigs here and n_msgs below?

Reply:
And sig64 adn msg32 are arrays of ptrys, pubkey is a direct array of objects.  That's a bit weird?  Also, please `pubkeys` to give the poor user a hint?